src/stores/courseData.js

```diff
@@ -240,6 +241,10 @@ const store = new Vuex.Store({
     setSubjectsInfo (state, data) {
       state.subjectsInfo = data;
     },
+    setUserYear (state, year) {
+      console.log('set user year to', year);
```

Comment:
Please remove logs before merging

Reply:
I'm mostly storing year because I need the year in a few places and this requires less converting back and forth. But maybe I should move `currentSemester` to the store and just calculate the year when I need it.

Reply:
Also I'm not actually sure how to convert from year to semester, which semester is 1/2/etc?

Reply:
Georgia explained this at the meeting, I will change this to just using Semester

Reply:
sounds good!